crash_similarity.py

```diff
@@ -50,10 +50,16 @@ def read_corpus(fnames):
         processed = preprocess(proto_signature)
 
         if frozenset(processed) not in already_selected:
-            elems.append((processed, data['signature']))
-        already_selected.add(frozenset(processed))
+            if embedding_algo == 'doc2vec':
```

Comment:
Yes, this would be nice, especially as we add more embedding algorithms after word2vec and doc2vec.